src/modules/submodules/typings.py

```diff
@@ -8,6 +9,7 @@ class MemberInfo:
     is_report_member: bool
     is_submit_member: bool
     is_fast_lane: bool
+    last_report_ref_slot: Optional[SlotNumber]
```

Comment:
Not sure it could be None
this is uint64

Reply:
fixed